pkg/fanal/handler/sysfile/filter.go

```diff
@@ -54,13 +54,15 @@ func newSystemFileFilteringPostHandler(artifact.Option) (handler.PostHandler, er
 // Handle removes files installed by OS package manager such as yum.
 func (h systemFileFilteringPostHandler) Handle(_ context.Context, result *analyzer.AnalysisResult, blob *types.BlobInfo) error {
 	var systemFiles []string
-	for _, file := range append(result.SystemInstalledFiles, defaultSystemFiles...) {
-		// Trim leading slashes to be the same format as the path in container images.
-		systemFile := strings.TrimPrefix(file, "/")
-		// We should check the root filepath ("/") and ignore it.
-		// Otherwise libraries with an empty filePath will be removed.
-		if systemFile != "" {
-			systemFiles = append(systemFiles, systemFile)
+	if result != nil {
```

Comment:
Do we need these changes?

Reply:
done at https://github.com/aquasecurity/trivy/pull/5439/commits/19fbf7b6f7d528e44d1cf720f78727c2956b4e27